cmd/operator-sdk/alpha/bundle/build.go

```diff
@@ -0,0 +1,103 @@
+package bundle
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/operator-framework/operator-registry/pkg/lib/bundle"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// newBundleBuildCmd returns a command that will build operator bundle image.
+func newBundleBuildCmd() *cobra.Command {
+	c := bundleCmd{}
+	cmd := &cobra.Command{
+		Use:   "build",
+		Short: "Build an operator bundle image",
+		Long: `The 'operator-sdk bundle build' command will build an operator
+bundle image containing operator metadata and manifests, tagged with the
+provided image tag. Metadata generated by this command defines this image
+and should not be modified.
+
+To write metadata and a bundle image Dockerfile to disk, set '--generate-only[=true]'
+Bundle metadata will be generated in <directory-arg>/metadata, and the Dockerfile
+in <directory-arg>. This flag is useful if you want to build the operator's
+bundle image manually.
+
```

Comment:
IHMO is missing described the purpose of the command. 
Where this metadata is used? When the user would like to use it?